Dockerfile.common.in

```diff
@@ -1,26 +1,16 @@
-FROM ${BASE_IMAGE_NAME}:${PHP_VERSION}
-
-LABEL authors="${AUTHOR}"
+FROM ${PHP_IMAGE_NAME}:${PHP_VERSION}
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 USER root
 
+# RUN echo 'deb http://ftp.us.debian.org/debian sid main' | tee -a /etc/apt/sources.list
+# hadolint ignore=DL3009
+RUN apt-get update
 # hadolint ignore=DL3008
-RUN echo 'deb http://ftp.us.debian.org/debian sid main' | tee -a /etc/apt/sources.list && \
-    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
-    apt-get install --no-install-recommends -y \
-      build-essential \
-      mysql-client-5.7 \
-      nodejs \
-      subversion \
-      && \
-    rm -rf /var/lib/apt/lists/* && \
-    docker-php-ext-install mysqli
-
-RUN npm install -g \
-      stylelint@${STYLELINT_VERSION} \
-      eslint@${ESLINT_VERSION}
+RUN apt-get install --no-install-recommends -y build-essential mariadb-client nodejs npm subversion
```

Comment:
Why do we need SVN? :thinking:

Reply:
We might also not need `mariadb-client` and `mysqli`, since these are unit tests?

Reply:
> Why do we need SVN?

https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/bin/install-wp-tests.sh#L107
😁

Reply:
> We might also not need `mariadb-client` and `mysqli`, since these are unit tests?

I see that some upstream tests actually require mysql 🤔 
eg. https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/2341/workflows/aa0fa2d3-24e1-43ac-a877-fcebc186da37/jobs/11935
`mysqladmin create wordpress_test --user=root --password= --host=127.0.0.1 --protocol=tcp`

Reply:
Yeah I think WordPress takes some liberty in what is considered a unit test. Probably also because most WP code doesn't lend itself to actual unit tests.